Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -101,6 +102,9 @@ class BrowserViewController: UIViewController {
     var navigationToolbar: TabToolbarProtocol {
         return toolbar ?? urlBar
     }
+    
+    var topTabsViewController: TopTabsViewController!
```

Comment:
Since `tabManager` is a let, you should be able to define this as a `lazy` property instead of an explicit unwrap and the later initialization down in `viewDidLoad`:

```swift
lazy var topTabsViewController = TopTabsViewController(tabManager: self.tabManager)
```

If that doesn't work you might need to wrap the initialization in a closure.